gemfiles/rails_4.1.gemfile

```diff
@@ -0,0 +1,24 @@
+# This file was generated by Appraisal
+
+source "http://rubygems.org"
+
+gem "appraisal"
+gem "rails", "~> 4.1.0"
+gem "rails-observers"
+gem "state_machine"
```

Comment:
@seuros That's actually not a typoo, original gem has a dependency the old state_machine, I've just use appraisal to test against both of them (the old state_machine and the new one state_machines), as you can see there is two of gemsets